local_settings.sh.in

```diff
@@ -303,8 +303,28 @@ uninstall_policies() {
 	_restore_file_modes
 }
 
+verify_policies() {
+	selinuxenabled || (echo "SELinux is disabled" && exit 0)
+	test $(getenforce) == Enforcing || (echo "SELinux is permissive" && exit 0)
```

Comment:
Are you sure about the parenthesis here? It doesn't seem to actually exit when I play with this line, the script seems to continue no matter what.

Reply:
meh, parenthesis are needed (else it would exit 0 in all cases), but create a subshell, so we're just exiting the subshell, not the main process.

Updated with a more conventional "if [... ] then ... fi" rather than the one-liner.